src/components/organisms/AuthenticationModal/RegisterForm/RegisterForm.tsx

```diff
@@ -164,7 +164,7 @@ const RegisterForm: React.FC<RegisterFormProps> = ({
       postRegisterCheck(auth, data);
 
       const accountProfileUrl = `/account/profile${
-        spaceId ? `?venueId=${spaceId}` : ""
+        spaceId ? `?spaceSlug=${spaceId}` : ""
```

Comment:
[NOTE]

This should be using one of those `url.ts` utility functions so we can avoid mess like this.